cpp/cmake/thirdparty/get_nvcomp.cmake

```diff
@@ -26,6 +26,7 @@ function(find_and_configure_nvcomp)
   if(TARGET nvcomp AND CUDF_USE_PER_THREAD_DEFAULT_STREAM)
     target_compile_definitions(nvcomp PRIVATE CUDA_API_PER_THREAD_DEFAULT_STREAM)
   endif()
+  target_link_libraries(nvcomp::nvcomp INTERFACE $<LINK_ONLY:CUDA::cuda_driver>)
```

Comment:
I think we can remove this with nvcomp 4.0.1, which fixed the hard linkage to `libcuda.so`.

Reply:
Yes we can